### PR TITLE
feat: adding shared-vpc support

### PR DESCRIPTION
**Pull request by adrianovs**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -120,3 +142,15 @@ variable "dns_servers" {
     error_message = "The variable 'var.dns_servers' should be an array with 1 or 2 DNS entries only."
   }
 }
+
+variable "create_ssh_fw_rule" {
```
Comment (MeNsaaH) [inline variables.tf:146]: Make this `create_ssh_firewall_rule`

File: versions.tf [hunk 2]
```diff
@@ -1,3 +1,10 @@
 terraform {
   required_version = ">= 0.14"
+  required_providers {
```
Comment (MeNsaaH) [inline versions.tf:3]: Remove the required providers here. This module is compatible with all versions of google so far

**Review by MeNsaaH — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (adrianoseie) [reply variables.tf]: Sure. done

**Comment by MeNsaaH** [comment]
This PR is included in version 2.4.0 :tada:

**Pull request merged**